src/modules/billing/services/charge-processor-service/lib/agreements.js

```diff
@@ -28,7 +28,7 @@ const mapAgreement = licenceAgreement => ({
 })
 
 const mapHistoryRow = row => {
-  const agreements = compact([row.section127Agreement, row.section130Agreement])
+  const agreements = [row.section127Agreement, row.section130Agreement].filter(x => !!x)
     .map(row => row.section127Agreement || row.section130Agreement)
```

Comment:
Since we're already put `.map()` onto a separate line we probably want `.filter()` on its own line too (also swapping the `!!x` for plain `x` as mentioned earlier):
```suggestion
  const agreements = [row.section127Agreement, row.section130Agreement]
    .filter(x => x)
    .map(row => row.section127Agreement || row.section130Agreement)
```